content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Updating configuration via a ConfigMap
+content_type: tutorial
```

Comment:
Great. Please also set a value for `weight` (should be different from all other weights within `content/en/docs/tutorials/configuration/`.

Reply:
Got it. I'll add a weight.

Reply:
Resolved.